Mathlib/Data/Sym/Card.lean

```diff
@@ -179,4 +179,21 @@ theorem card_subtype_not_diag [Fintype α] :
 protected theorem card [Fintype α] : card (Sym2 α) = Nat.choose (card α + 1) 2 :=
   Finset.card_sym2 _
 
+variable {a b : α}
+
+theorem univ_singleton : @univ (Sym2 ({a} : Finset α)) _ = {s(⟨a, by simp⟩, ⟨a, by simp⟩)} := by
+  ext p
+  simp_rw [mem_univ, mem_singleton, true_iff]
+  refine p.inductionOn fun ⟨x, hx⟩ ⟨y, hy⟩ ↦ ?_
+  rw [mem_singleton] at hx hy
+  simp [hx, hy]
+
+theorem univ_pair : @univ (Sym2 ({a, b} : Finset α)) _ = {s(⟨a, by simp⟩, ⟨a, by simp⟩),
+    s(⟨a, by simp⟩, ⟨b, by simp⟩), s(⟨b, by simp⟩, ⟨b, by simp⟩)} := by
```

Comment:
When the type of your statement spans several lines, please start it on a new line. Same elsewhere
```suggestion
theorem univ_pair :
    @univ (Sym2 ({a, b} : Finset α)) _ = {s(⟨a, by simp⟩, ⟨a, by simp⟩),
      s(⟨a, by simp⟩, ⟨b, by simp⟩), s(⟨b, by simp⟩, ⟨b, by simp⟩)} := by
```